DataFormats/L1THGCal/interface/HGCalClusterT.h

```diff
@@ -134,6 +134,46 @@ namespace l1t
       uint32_t subdetId() const {return detId_.subdetId();} 
       uint32_t layer() const {return detId_.layer();}
       int32_t zside() const {return detId_.zside();}
+
+      int32_t ClusterET()             const {return ClusterET_;}
```

Comment:
I am wondering if there is a possibility to use these variables in the HGCalMulticluster as well. The purpose would be to directly decode the HGCalCluster into the Multicluster object.